core/src/main/java/com/linecorp/armeria/server/ServerBuilder.java

```diff
@@ -966,6 +987,17 @@ public ServerBuilder annotatedService(String pathPrefix, Object service,
         return annotatedService(pathPrefix, service, Function.identity(), exceptionHandlersAndConverters);
     }
 
+    /**
+     * Binds the specified annotated service object under the path prefix {@code "/"}.
+     *
+     * @param customizer the {@link AnnotatedHttpServiceConfiguratorSetters}.
```

Comment:
This description is different from the one above at line 953 even though it's the same one.